pkg/provisioner/crdprovisioner.go

```diff
@@ -237,7 +237,7 @@ func (c *CrdProvisioner) CreateVolume(
 	if err != nil {
 		go func() {
 			conditionFunc := func() (bool, error) {
-				if err := azV.Delete(ctx, azVolumeName, metav1.DeleteOptions{}); err != nil {
+				if err := azV.Delete(context.TODO(), azVolumeName, metav1.DeleteOptions{}); err != nil {
```

Comment:
`context.Background()` would be appropriate here since this is a background task.